src/app.js

```diff
@@ -48,6 +48,20 @@ import {
 
 import { registerOnDeeplinkSet } from "../global/globalVariables"
 
+function Scene(props) {
```

Comment:
Can we use:
```
const Scene = (Platform.OS == 'Lumin') ? view : scene;
const Prism = (Platform.OS == 'Lumin') ? view : prism;
```
instead?

Reply:
I don't see much difference as it's temporary fix till we'll be able to define correct exports in `magic-script-components` and `magic-script-components-lumin`. As we discussed during the call I tired this solution and I received:

![Simulator Screen Shot - iPhone 11 Pro - 2020-03-05 at 12 28 28](https://user-images.githubusercontent.com/42351184/75977700-44023480-5edd-11ea-8ca6-59fea2c92f15.png)

it's probably due to lack of import for scene and prism from `magic-script-component-react-native` which we have to avoid in app.js (maybe those imports should be `MagicScript.js` but it looks like too much to introduce the workaround).